tests/tests/tier0/bluechi-change-port-with-c-cmd-option/test_bluechi_change_port_with_c_cmd_option.py

```diff
@@ -0,0 +1,66 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+from typing import Dict
+import os
+import time
+
+from bluechi_test.test import BluechiTest
+from bluechi_test.container import BluechiControllerContainer, BluechiNodeContainer
+from bluechi_test.config import BluechiControllerConfig, BluechiNodeConfig
+from bluechi_test.util import read_file
+
+NODE_FOO = "node-foo"
+
+
+def check_if_node_and_ctrl_connected(ctrl, node):
+    assert os.popen("lsof -i:8421").read()
+    assert "Registered managed node from" in ctrl.exec_run("systemctl status bluechi-controller.service")[1]
+    assert f"Connected to manager as '{NODE_FOO}'" in node.exec_run("systemctl status bluechi-agent.service")[1]
+
+
+def check_if_node_disconnected(node):
+    time.sleep(1)
```

Comment:
Not sure that it's the best option, But it takes time till it disconnects from port 8420